spec/helpers/simple_server_env_helper_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe SimpleServerEnvHelper do
+  include SimpleServerEnvHelper
+
+  let(:simple_server_env) {"SIMPLE_SERVER_ENV"}
+
+  after(:each) do
+    ENV[simple_server_env] = "test"
+  end
+
+  describe 'style_class_for_environment' do
+    it 'should return the correct style class for the environment' do
+      ENV[simple_server_env] = "qa"
+
+      style = style_class_for_environment
+      puts style
```

Comment:
Don't leave stray `puts` in the spec.

Reply:
Removed the `puts` and the redundant var.